File: Java/Java/CaseTecnico/src/classes/Pessoa.java
package classes;

public class Pessoa {

	private String nome;
	private int idade;

	public Pessoa(String nome, int idade) {
		this.nome = nome;
		this.idade = idade;
	}

	public void imprimirPessoa() {
		System.out.println("Nome = " + this.nome);
		System.out.println("Idade = " + this.idade + "\n");
	}

	public int getIdade() {
		return this.idade;
	}

	public String getNome() {
		return this.nome;
	}
}

File: CSharp/CaseTecnico/CaseTecnico/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CaseTecnico
{
    public class Pessoa
    {
        public string nome { get; set; }
        public int idade { get; set; }

        public Pessoa(string nome, int idade)
        {
            this.nome = nome;
            this.idade = idade;
        }

        public void imprimirPessoa()
        {
            Console.WriteLine("Nome = " + this.nome);
            Console.WriteLine("Idade = " + this.idade + "\n");
        }
    }
}
